README.md

```diff
@@ -95,6 +100,47 @@ User.create(
 )
 ```
 
+## Troubleshooting
+
+If you are getting errors such as `FATAL: role "user" doesn't exist. Couldn't create database.` while running `rake db:setup` or `rake db:create` then it may mean that your database is password protected. There are two ways to setup required databases:
+  
+1. (Recommended) Loosen local Postgres database security to allow local users without password
```

Comment:
Ok, I may have been a little unclear in other discussions. Sorry for that. I really don’t think we should ever recommend people reduce the security of their existing setup. (I don’t think security is important for a dev DB here, but if someone has any other databases in Postgres that might actually need to be secured, we’re recommending they open up a big vulnerability. That’s bad.)

Can we make this option 2 and make the other (manually create users and DBs) the recommended option?